app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/package.scala

```diff
@@ -40,11 +42,13 @@ package object controllers {
       }
     }
 
-    private val nonEmptyTelephoneNumber: Constraint[String] = Constraint[String] { fieldValue: String =>
```

Comment:
the old name, `nonEmptyTelephoneNumber`, seems more descriptive than `telephoneNumberConstraint`